local.js

```diff
@@ -103,7 +105,11 @@ function requestTokenDeduped(scope) {
 	return IN_FLIGHT_REQUESTS[scope];
 }
 
-module.exports = function getLocalJwt(scope) {
+module.exports = function getLocalJwt(scope, doNotKeepSessionAlive) {
```

Comment:
Can we make the second parameter an options object instead (just call it `opts` here)

Reply:
Yes, I'll make that change!